template.json

```diff
@@ -1,18 +0,0 @@
-{
```

Comment:
@VerFacilLtd no, please, don't rename the `template.json` to create yours, copy it instead, otherwise we will be left without template ;) Please restore it and I'll merge the pull request.

Reply:
Sorry but I am new to this. How do I restore the template?

Do I close the pull request and start again?